doc/source/user_guide/tutorials/index.rst

```diff
@@ -3,94 +3,143 @@
 Tutorials
 ---------
 
+What do you need to know
+++++++++++++++++++++++++
 
-The following tutorials show in details how to use basic features of PyDPF-Core
-for each usage main step.
+This tutorials will guide you through the steps required to access, analyze,
+and transform simulation data using  PyDPF-Core.
 
-:bdg-dark-line:`Define simulation data` :octicon:`arrow-right;1em` :bdg-dark-line:`Store input data in DPF` :octicon:`arrow-right;1em` :bdg-dark-line:`Transform available data` :octicon:`arrow-right;1em` :bdg-dark-line:`Extract data`
+The tutorials are more substantive and complex than examples found in the standard
+documentation set. They are designed to teach how to perform a task and understand
+some concepts, providing detailed explanations at each stage.
```

Comment:
```suggestion
Tutorials are more substantive and complex than examples. They are designed to teach how to perform a task and understand
the underlying concepts, providing detailed explanations at each stage, whereas examples showcase end-to-end specific processes.
```

Reply:
Not sure if tutorials can be said to be more `complex` than examples.

Reply:
more elaborated ?